react/tic-tac-toe/src/utils/Validations/index.js

```diff
@@ -0,0 +1,20 @@
+// eslint-disable-next-line no-useless-escape
+const VALID_EMAIL = /^\w+([\.-]?\w+)*@wolox.com/;
+
+const isRequired = value => value ? undefined : 'Required field';
+
+const hasLength = value =>
+  value.length < 8 ? 'Value length has to be greater than 8 characters' : undefined;
```

Comment:
Same and remember to do `value && value.length`. Always think what could happen if you get undefined instead of a value. 

Your functions should always be capable of handle everything and don't break in the process